src/components/Contacts/ContactFilters/ContactFilters.tsx

```diff
@@ -0,0 +1,44 @@
+import { QueryLazyOptions } from '@apollo/client';
+import { colors } from '@material-ui/core';
+import React from 'react';
+import { Exact } from '../../../../graphql/types.generated';
+import { ContactFiltersQuery } from '../../../../pages/accountLists/[accountListId]/ContactFilters.generated';
+
+interface Props {
+  data: ContactFiltersQuery;
+  loading: boolean;
+  error: Error;
+  loadFilters: (
+    options?: QueryLazyOptions<Exact<{ accountListId: string }>>,
+  ) => void;
+}
+
+const ContactFilters: React.FC<Props> = ({
+  data,
+  loading,
+  error,
+  loadFilters,
+}: Props) => {
+  return (
+    <div style={{ backgroundColor: colors.amber[600] }}>
+      <h2>Filters</h2>
+      <button data-testID="LoadFiltersButton" onClick={() => loadFilters()}>
+        Load Filters
+      </button>
+      {error && <p data-testID="ErrorText">Error: {error.toString()}</p>}
```

Comment:
I think you can use [`getByText`](https://testing-library.com/docs/queries/bytext) here.
You should be able to use an `exact: false` match or a regex to match the beginning of the string https://testing-library.com/docs/queries/about#textmatch